takwimu/takwimu_ui/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Unless there is a specific reason for it, I'd rather we use the a single `.gitignore` file for the project (and we've already included `node` there).